content/automate/ManageIQ/System/Event/EmsEvent/EMBEDDED_ANSIBLE.class/__class__.yaml

```diff
@@ -0,0 +1,433 @@
+---
+object_type: class
+version: 1.0
+object:
+  attributes:
+    description: 
+    display_name: Ansible Tower
+    name: ANSIBLE_TOWER
```

Comment:
I do not think we need to duplicate the information.  Can we use this:
```yaml
  display_name:
  name: Embedded_Ansible
```

The duplications and upper-case make it look odd
![image](https://cloud.githubusercontent.com/assets/2591569/23728388/84bd0fe2-042a-11e7-920e-a7ed8bbdb600.png)

I would prefer to just have:
![image](https://cloud.githubusercontent.com/assets/2591569/23728396/9362e47c-042a-11e7-8d7a-6acb7957b1b3.png)

As for external Ansible that is distinguished by the "Ansible_Tower" name.